tests/integration/src/test/java/org/apache/pulsar/tests/integration/containers/StandaloneContainer.java

```diff
@@ -31,29 +31,37 @@ public class StandaloneContainer extends PulsarContainer<StandaloneContainer> {
     public static final String NAME = "standalone";
 
     public StandaloneContainer(String clusterName) {
-        super(clusterName,
-            NAME,
-            NAME + "-cluster",
-            "bin/pulsar",
-            BROKER_PORT,
-            BROKER_HTTP_PORT);
+        this(clusterName, DEFAULT_IMAGE_NAME);
     }
 
     public StandaloneContainer(String clusterName, String pulsarImageName) {
+        this(clusterName, pulsarImageName, false);
+    }
+
+    public StandaloneContainer(String clusterName, String pulsarImageName, boolean enableTls) {
         super(clusterName,
                 NAME,
                 NAME + "-cluster",
-                "bin/pulsar",
+                pulsarImageName.endsWith("latest") ? "bin/run-standalone.sh" : "bin/pulsar",
```

Comment:
Why do we need to check if the image name ends with "latest"?

Reply:
Because the script does't support `standalone` in the previous shell after this change.
See this error:
https://github.com/apache/pulsar/actions/runs/6270304613/job/17028516873